app/services/setToken.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../db');
+const sql = require('../db/sql.js');
+
+module.exports = async (username, token) => {
+  return await db.query(sql`
+    insert into sessions(token, created_at, user_id)
+    values (${token}, current_timestamp,
+      (select id from users where username = ${username})
+    )
+  `);
```

Comment:
also, one other thing aside from the `await` thing, you might want to leverage JS's implicit `return` when you only do one thing:
```js
module.exports = (username, token) => (  // <-- Notice I used () for having a multi-line one-liner
  db.query(sql`
    insert into sessions(token, created_at, user_id)
    values (${token}, current_timestamp,
      (select id from users where username = ${username})
    )
  `);
);
```